tests/acceptance/bootstrap/SharingNgContext.php

```diff
@@ -509,16 +509,33 @@ public function userSendsTheFollowingSpaceShareInvitationToFederatedUserUsingPer
 	}
 
 	/**
-	 * @When user :user updates the last resource share with the following using the Graph API:
+	 * @Given user :user has updated the last resource share with the following properties:
+	 *
+	 * @param string $user
+	 * @param TableNode $table
+	 *
+	 * @return void
+	 */
+	public function userHasUpdatedTheLastResourceShareWithTheFollowingProperties(string $user, TableNode $table): void {
+		$permissionID = $this->featureContext->shareNgGetLastCreatedUserGroupShareID();
+		$response = $this->updateResourceShare(
+			$user,
+			$table,
+			$permissionID
+		);
+		$this->featureContext->theHTTPStatusCodeShouldBe(200, "Expected response status code should be 200", $response);
+	}
+
+	/**
+	 * @When user :user updates the last resource share with the following properties using the Graph API:
 	 *
 	 * @param string $user
 	 * @param TableNode $table
 	 *
 	 * @return void
 	 */
 	public function userUpdatesTheLastShareWithFollowingUsingGraphApi($user, TableNode $table) {
```

Comment:
same goes for this. please check if the same review applies to other lines as well.
Reviewer just highlights one line but that should also be applied to other lines.
```suggestion
	public function userUpdatesTheLastShareWithFollowingPropertiesUsingGraphApi($user, TableNode $table) {
```

Reply:
Got you :+1: